### PR TITLE
Washer page before/after job photos

### PR DESCRIPTION
**Pull request by joelgroomer**
# Description

Allows washers to see their current job on the map and to submit before and after photos.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Change Status

Still waiting on some back-end updates to make sure this works 100%. Also some rough edges need to be smoothed out.

# How Has This Been Tested?

- [x] Manual testing

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: ExpressWash/ExpressWash/CarController.swift [hunk 1]
```diff
@@ -60,16 +60,44 @@ class CarController {
...
+            car.year = rep.year
+            do {
+                try privateMOC.save()
+            } catch {
+                print("Unable to save updated car: \(error)")
+                context.reset()
+                completion(nil, error)
             }
+            completion(car, nil)
         }
```
Comment (Keffury1) [inline ExpressWash/ExpressWash/CarController.swift:101]: This function can go as discussed

**Review by Keffury1 — CHANGES_REQUESTED** [review]
It all looks great. Requesting a change on the car controller.

File: ExpressWash/ExpressWash/CarController.swift (hunk 1 above)
Comment (joelgroomer) [reply ExpressWash/ExpressWash/CarController.swift:101]: Changes didn't work correctly. Leaving as-is for now. Everything is working and we can change if needed.

**Review by joelgroomer — COMMENTED** [review]
Please see individual comment below.

**Review by Keffury1 — APPROVED** [review]

**Pull request merged**